forem/forem
# Implement new color picker in listings category form, cleanup old code

## What type of PR is this? (check all applicable)


## Description

This PR finishes off the final tasks for https://github.com/forem/forem/issues/15179:

- introduces the new picker in the listings category form (in `/admin/apps/listings/categories/${categoryId}/edit`)
- Deletes and old initializer no longer used
- Deletes old CSS not longer used

NB: the final two points assume the following PRs are merged before this one:
- https://github.com/forem/forem/pull/16769
- https://github.com/forem/forem/pull/16768

## Related Tickets & Documents



- Closes https://github.com/forem/forem/issues/15179

## QA Instructions, Screenshots, Recordings

- Locally edit a listing category (at `/admin/apps/listings/categories/${categoryId}/edit`) - check the color picker works as expected

### UI accessibility concerns?

Makes the listing category color picker accessible 🙂 

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- Overarching issue that this closes is tagged for the changelog rollup


## [optional] What gif best describes this PR or how it makes you feel?

![rainbow unicorn](https://media.giphy.com/media/26AHG5KGFxSkUWw1i/giphy.gif)
